pcweb/pages/docs/__init__.py

```diff
@@ -112,6 +113,21 @@ def exec_blocks(doc, href):
 outblocks = []
 
 
+manual_titles = {
+    "docs/library/datadisplay/callout-ll.md": "Callout Low",
```

Comment:
I think in the place where we set the low level, we can just modify the name, then it will apply to all Low Level components.

Reply:
changed